spec.bs

```diff
@@ -1014,23 +1014,28 @@ special value "*". The only interfaces that can be exposed inside of the globals
 specification are those that explicitly list the global names provided here.
 
 <pre class="idl">
-[Exposed=InterestGroupBiddingScriptRunnerGlobalScope,
-Global=InterestGroupBiddingScriptRunnerGlobalScope]
-interface InterestGroupBiddingScriptRunnerGlobalScope {
+[Global=InterestGroupScriptRunnerGlobalScope, Exposed=InterestGroupScriptRunnerGlobalScope]

```

Comment:
I might stick with a new line here as if you look in the preview, it's a bit hard to read (as with some of the instances below too)

Reply:
Done. Also broke the interface line to two lines.